hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelinePlacementPolicy.java

```diff
@@ -195,9 +200,11 @@ public void testChooseNodeNotEnoughSpace() throws SCMException {
     int nodesRequired = HddsProtos.ReplicationFactor.THREE.getNumber();
 
     thrownExp.expect(SCMException.class);
-    thrownExp.expectMessage("enough space for even a single container");
+    thrownExp.expectMessage("Unable to find enough nodes that meet" +
+        "the space requirement");
```

Comment:
Here should a single space, then the failed ut should pass I think.

Reply:
Thanks, I wrapped the line for checkstyle and did not re-run the test after.